.gitignore

```diff
@@ -20,7 +20,6 @@ coverage
 # misc
 package-lock.json
 .DS_Store
-.vscode
```

Comment:
Can remove this as it was added in #609

Reply:
Just checking - are you saying that this change shouldn't be in the PR, as it was part of a previously merged PR (and we should just rebase)? Or are you saying that you want `.vscode` to be in `.gitignore`?

Reply:
👇
>  this change shouldn't be in the PR, as it was part of a previously merged PR